vllm/worker/hpu_model_runner.py

```diff
@@ -3823,13 +3832,13 @@ def try_revert_dummy_output_tokens():
                                 list(self.graphed_multimodal_buckets)
                             # set is unhasable and causes friction with
                             # hpu graphs, hence turning it to a list
-                        execute_model_kwargs = \
-                            self.model.compute_input_embeddings_for_mrope_mm_optimized(
+                            execute_model_kwargs = self.model.compute_input_embeddings_for_mrope_mm_optimized(
```

Comment:
is this a bug fixing or accident change? move `execute_model_kwargs=self.model.compute_input_embeddings_for_mrope_mm_optimized ` under condition `if 'pixel_values' in execute_model_kwargs and \
                                self.is_mm_optimized:`